### ISSUE TITLE
Full re-write required from Redactor v.9 to v.10 - Button doesn't display

### ISSUE THREAD

**Issue by Guilm** [Natetronn/redactor-css#1]
Hi, i have installed the plugin and add the parameter in the JSON.

However I don't see the CSS3 button in the editor field toolbar. Can you help me ? :)

My version of Craft : 2.3.2629

Thanks for your help !

**Comment by Natetronn** [comment]
Hi Guilm, the whole plugin will need to be re-written for v.9 to v.10 API updates. I've started though, it's a full re-write so I'm not sure when I'll get it up. I'll let you know as soon as it's up, however.

**Comment by Natetronn** [comment]
Note to all: currently redactor-css works with Craft 2.2.2607 and below. Craft 2.3.2615 introduces Redactor v.10 and where the re-write is required.

**Comment by richardgorman** [comment]
Hi, are you still continuing with the development of this plugin?

**Comment by Natetronn** [comment]
tl;dr I don't think so.

It really was never much more than a beta plugin to begin with as it needed more work. Allot of people are still using Redactor 1 as a separate plugin even though Craft core comes with Redactor 2. And Redactor 2's API changed quite a bit. I'm not sure I want to support two versions of this plugin. I think there are some other options available as well.

Maybe take a look [here](http://craftcms.stackexchange.com/questions/10834/where-do-i-add-custom-css-for-the-redactor-editor-in-the-craft-control-panel).
